deployments/managed-services.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Managed services"
+---
+
+One of the key features of Architect.io is its dependency graph, which is a visual representation of the relationships between different services in an application.
+The dependency graph is a useful tool for managing complex applications with multiple services and dependencies, and demonstrates how those services work together in the application.
+
+Let's consider an example of a simple web application with a single `hello-world` service.
+The dependency graph for this application would have two nodes: one for the `hello-world` service and one for the Traefik `gateway`.
+
+![](/images/dependency-graph.png)
+
+# Gateway Node (traefik)
+The top node is a gateway node that uses Traefik to route traffic to the service. In this example, the gateway node would depend on the `hello-world` service, as it needs to route traffic to it.
```

Comment:
It would be good to include a link to both Traefik and cert-manager to give people more context